Extension/src/LanguageServer/extension.ts

```diff
@@ -70,6 +70,17 @@ export function activate(activationEventOccurred: boolean): void {
         return;
     }
 
+    let errMsg: string;
```

Comment:
I thought we were going to put these changes in main.ts so that debugging will be disabled too.

Reply:
I didn't realize there was an activation in main.ts. Originally the message was in client.ts and then I moved it higher up in extension.ts so I thought that was it. True, activate in main.ts is the correct place. I made the correction, retested and pushed.